.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 
-python: "3.5"
+python: "3.7"
```

Comment:
this doesn't affect the tested python version, because evaluations happen in docker not in travis